betterconf/caster.py

```diff
@@ -11,15 +11,18 @@ def cast(self, val: str):
 
 class ConstantCaster(AbstractCaster, typing.Generic[VT]):
 
-    ABLE_TO_CAST: typing.Dict[str, typing.Any] = {}
+    ABLE_TO_CAST: typing.Dict[typing.Union[str, typing.Tuple[str]], typing.Any] = {}
```

Comment:
typing.Tuple[str] is ("one arg",). I think you wanted `typing.Tuple[str, ...]` that has non-constant count of args

Reply:
Yes, I wanted exactly that. This is fixed too.